src/reflect/scala/reflect/internal/SymbolTable.scala

```diff
@@ -432,6 +444,9 @@ abstract class SymbolTable extends macros.Universe
   implicit val StringContextStripMarginOps: StringContext => StringContextStripMarginOps = util.StringContextStripMarginOps
 }
 
-object SymbolTableStats {
+trait SymbolTableStats {
+  self: scala.reflect.internal.TypesStats =>
   val phaseCounter = Statistics.newCounter("#phase calls")
+  // Defined here because `SymbolLoaders` needs it and has no access to global
+  val classReadNanos = Statistics.newSubTimer("time classfilereading", typerNanos)
```

Comment:
Unfortunately, `SymbolLoaders`'s statistics comes from the `SymbolTable`, not `Global`, so this had to be moved here because `statistics` at this point hasn't been overridden and only contains the timers and counters from Scala reflect.

Reply:
You mean it's not ideal because `SymbolLoaders` is defined in `scala.tools.nsc` and not `scala.reflect`? Maybe add that to the comment. But it's no big deal, really.

Reply:
Yeah, exactly. I'll add it. But good to know it's not a big deal. My goal was to avoid moving the definition of statistics elsewhere, so I was a little bit disappointed with this :smile:.